### DISCUSSION TITLE
Issues running local dev enviroment

### DISCUSSION THREAD

**Discussion by melanki** [Budibase/budibase#3708] [Help]
When using `yarn setup & yarn build` as described in CONTRIBUTING.md I get the issue:

> Redis connection failed - Error: connect ECONNREFUSED 127.0.0.1:6379

I am guessing `yarn setup & yarn build `does not start the Redis docker?

**Comment by mjashanks** [comment]
Hi there.

There is an issue in the scripts, which we will have fixed.

In the meantime, you should be able to get started by
1. in the `hosting` directory, there is a `hosting.properties` file - make a copy, and call it `.env`
2. go back to the root dir, and run `yarn setup`

`yarn setup` should start all the necessary containers.
